core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -647,6 +648,25 @@ public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMa
     return result;
   }
 
+  private Object getTypedValue(Map.Entry<String, String> entry, PropertyKey key) {
```

Comment:
shall we put this into Configuration related classes? It looks useful to me

Reply:
Yeah